### DISCUSSION TITLE
upload document error

### DISCUSSION THREAD

**Discussion by bigbenz8** [open-webui/open-webui#1159] [General]
# Bug Report

## Description
when I upload a .pdf file, after some seconds ... an error show : Something went wrong :/\ncannot reshape array of size 22356 into shape (207,216,newaxis)

**Bug Summary:**
![image](https://github.com/open-webui/open-webui/assets/8120813/bc861e9a-e91e-4256-aeaf-dae4fb4b0261)

## Environment

- **Operating System:** [Ubuntu 22.04 (win10 - wsl2 )]

**Comment by tjbck** [comment]
Related: https://github.com/open-webui/open-webui/discussions/1095

Try turning off the OCR feature from document settings!

**Comment by bigbenz8** [comment]
> Related: #1095
> 
> Try turning off the OCR feature from document settings!

Thank you. However, I can't seem to find a way to turn off the OCR feature in Document Settings, can you guide me? 
![image](https://github.com/open-webui/open-webui/assets/8120813/54033321-f555-4a11-94ab-5733f431cff2)

**Comment by justinh-rahb** [comment]
You're running an out of date version, would suggest updating, there's fixes to bugs relating to RAG.

**Comment by dr-kd** [comment]
I just had this same probem with a new build from  commit 30b053116d6a and release v0.1.123

My workaround was to apt instlal ocrmypdf onto another machine and use that to add the text layer prior to presenting it to openwebui.

**Comment by juanjgit** [comment]
This is still an issue in v0.3.12
